coordinator/src/db/positions.rs

```diff
@@ -67,6 +67,26 @@ impl Position {
         Ok(positions)
     }
 
+    #[autometrics]
```

Comment:
Why do we need `autometrics` for this call?

Reply:
copy paste... we have this on all the functions inside of this `Node` impl block at the moment. I'm happy to remove it (or add it to the function below as well where I did not add it 😅)